p2p/event_handler.go

```diff
@@ -40,6 +41,31 @@ func (m *p2pModule) HandleEvent(event *anypb.Any) error {
 			}
 		}
 
+	case messaging.StateMachineTransitionEventType:
+		stateMachineTransitionEvent, ok := evt.(*messaging.StateMachineTransitionEvent)
+		if !ok {
+			return fmt.Errorf("failed to cast event to StateMachineTransitionEvent")
+		}
+
+		if stateMachineTransitionEvent.NewState == string(coreTypes.StateMachineState_P2P_Bootstrapping) {
+			addrBook := m.network.GetAddrBook()
+			if len(addrBook) == 0 {
+				m.logger.Warn().Msg("No peers in addrbook, bootstrapping")
+
+				err := bootstrap(m)
```

Comment:
https://github.com/pokt-network/pocket/pull/491/files#r1108018053

```suggestion
				if err := bootstrap(m); err != nil {
```